configs/bedrock-prod.env

```diff
@@ -23,7 +23,7 @@ SWITCH_DOUBLE_CLICK=on
 SWITCH_EXPERIMENT_FIREFOX_FAMILY=off
 SWITCH_EXPERIMENT_FIREFOX_NEW_ALL_LINKS_PT2=off
 SWITCH_EXPERIMENT_FIREFOX_NEW_UP_TO_DATE_USERS=off
-SWITCH_EXPERIMENT_FIRSTRUN_MEMBERSHIP=on
+SWITCH_EXPERIMENT_FIRSTRUN_MEMBERSHIP=off
```

Comment:
Should we just delete this switch?

Reply:
Might as well. I didn't know enough about this test but if they want to turn it back on we can just unbalete. 🥊